src/test/java/com/flightstats/hub/webhook/WebhookCoordinatorTest.java

```diff
@@ -201,11 +201,11 @@ void testIfNotLeaderWontRegisterServices() {
         services.forEach((type, svcs) -> assertTrue(type + " has services registered", svcs.isEmpty()));
     }
 
-    private WebhookManager getWebhookManager() {
-        return new WebhookManager(
-                localWebhookManager,
+    private WebhookCoordinator getWebhookManager() {
```

Comment:
Does it make sense to rename this getter to getWebhookCoordinator?

Reply:
Yep, will do!